### PR TITLE
Add ability to call molecule as a python module

### PR DESCRIPTION
**Pull request by ssbarnea**
Fixes: #1522

### CODE REVIEW

**Comment by gundalow** [comment]
Thanks for the PR. Wonder if/where this should be in the docs as well?

File: molecule/__main__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+#  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+#  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+#  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+#  FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+#  DEALINGS IN THE SOFTWARE.
+
+from .shell import main
+
+
+main()
```
Comment (webknjaz) [inline molecule/__main__.py]:
```suggestion
if __name__ == '__main__':
    main()
```

File: doc/source/installation.rst [hunk 2]
```diff
@@ -54,6 +54,17 @@ Install Molecule:
 .. code-block:: bash
 
     $ sudo pip install molecule
+    # non root installation:
+    $ pip install --user molecule
```
Comment (webknjaz) [inline doc/source/installation.rst]: Maybe just leave this non-sudo line?

File: molecule/__main__.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+#  Copyright (c) 2015-2018 Cisco Systems, Inc.
```
Comment (webknjaz) [inline molecule/__main__.py]: s/Cisco/Red Hat/

**Comment by ssbarnea** [comment]
Sorry, GitHub decided by itself to close this PR so I ended up creating a new one after switching to a feature branch. Lets do it at https://github.com/ansible/molecule/pull/1545 -- i fixed the comments.

**Comment by webknjaz** [comment]
Oh, you @ssbarnea probably force pushed the PR-bound branch in your fork, which was master to match upstream, which rendered a diff with zero changes. It's probably the reason GH closed it. FTR.

**Pull request closed without merging**